packages/synapse-react-client/src/components/EntityFinder/details/view/DetailsView.tsx

```diff
@@ -78,36 +109,271 @@ export type EntityFinderTableViewRowData = EntityFinderHeader & {
   currentSelectedVersion?: number
 }
 
+const columnHelper = createColumnHelper<EntityFinderTableViewRowData>()
+
+function NameRenderer(props: {
+  context: CellContext<EntityFinderTableViewRowData, string>
+  setCurrentContainer?: (containerId: string) => void
+}) {
+  const { context, setCurrentContainer } = props
+  const { row, getValue } = context
+  const rowName = getValue()
+  if (setCurrentContainer && isContainerType(row.original.entityType)) {
+    return (
+      <span
+        role="link"
+        className="EntityFinderTableCellContainerLink"
+        onClick={e => {
+          e.stopPropagation()
+          setCurrentContainer(row.original.id)
+        }}
+      >
+        <Tooltip title={rowName}>
+          <span className="nameColumnCell">{rowName}</span>
+        </Tooltip>
+      </span>
+    )
+  } else {
+    return (
+      <Tooltip title={rowName}>
+        <span className="nameColumnCell">{rowName}</span>
+      </Tooltip>
+    )
+  }
+}
+
+function SelectAllCheckboxRenderer(props: {
+  isVisible: boolean
+  disabled: boolean
+  checked: boolean
+  onSelectAll: () => void
+}) {
+  const { isVisible, disabled, checked, onSelectAll } = props
+  if (!isVisible) {
+    return null
+  }
+
+  return (
+    <Checkbox
+      inputProps={{ 'aria-label': 'Select All' }}
+      checked={checked}
+      disabled={disabled}
+      onChange={() => {
+        if (!disabled) {
```

Comment:
Surprised you need to check for this, but it certainly doesn't hurt!

Reply:
I can remove it--I think this is an artifact of some refactoring. This used to be wrapped in a div that had the handler, which had to check the `disabled` state.